packages/vuetify/src/components/VChip/VChip.tsx

```diff
@@ -190,46 +194,68 @@ export const VChip = genericComponent<VChipSlots>()({
           { genOverlays(isClickable.value, 'v-chip') }
 
           { hasFilter && (
-            <VDefaultsProvider
-              key="filter"
-              defaults={{
-                VIcon: { icon: props.filterIcon },
-              }}
-            >
-              <VExpandXTransition>
-                <div
-                  class="v-chip__filter"
-                  v-show={ group.isSelected.value }
-                >
-                  { slots.filter ? slots.filter() : (<VIcon />) }
-                </div>
-              </VExpandXTransition>
-            </VDefaultsProvider>
+            <VExpandXTransition key="filter">
+              <div
+                class="v-chip__filter"
+                v-show={ group.isSelected.value }
+              >
+                { !slots.filter ? (
+                  <VIcon
+                    key="filter-icon"
+                    icon={ props.filterIcon }
+                  />
+                ) : (
+                  <VDefaultsProvider
+                    key="filter-defaults"
+                    disabled={ !props.filterIcon }
+                    defaults={{
+                      VIcon: { icon: props.filterIcon },
+                    }}
+                    v-slot:default={ slots.filter }
+                  />
+                )}
+              </div>
+            </VExpandXTransition>
           )}
 
           { hasPrepend && (
-            <VDefaultsProvider
-              key="prepend"
-              defaults={{
-                VAvatar: {
-                  image: props.prependAvatar,
-                },
-                VIcon: {
-                  icon: props.prependIcon,
-                },
-              }}
-            >
-              { slots.prepend
-                ? (
-                  <div class="v-chip__prepend">
-                    { slots.prepend() }
-                  </div>
-                )
-                : props.prependAvatar ? (<VAvatar start />)
-                : props.prependIcon ? (<VIcon start />)
-                : undefined
-              }
-            </VDefaultsProvider>
+            <div key="prepend" class="v-chip__prepend">
+              { !slots.prepend ? (
+                <>
+                  { props.prependIcon && (
+                    <VIcon
+                      key="prepend-icon"
+                      icon={ props.prependIcon }
+                      start
+                    />
+                  )}
+
+                  { props.prependAvatar && (
+                    <VAvatar
+                      key="prepend-avatar"
+                      icon={ props.prependAvatar }
+                      start
+                    />
+                  )}
+                </>
+              ) : (
+                <VDefaultsProvider
+                  key="prepend-defaults"
+                  disabled={ !hasPrependMedia }
```

Comment:
disabled means that start isn't provided to the slot any more

before:
![Screenshot_20230309_011512](https://user-images.githubusercontent.com/16421948/223735861-db8de3d5-fa95-495a-a6eb-7a3e11f8140d.png)
after:
![Screenshot_20230309_011432](https://user-images.githubusercontent.com/16421948/223735741-47a71990-206d-4837-9986-7c0c071e0b48.png)

```html
<v-chip prepend-avatar="https://cdn.vuetifyjs.com/images/parallax/material.jpg" />
<v-chip>
  <template #prepend>
    <v-avatar image="https://cdn.vuetifyjs.com/images/parallax/material.jpg" />
  </template>
</v-chip>
<v-chip>
  <template #prepend>
    <v-avatar>
      <v-img src="https://cdn.vuetifyjs.com/images/parallax/material.jpg" cover />
    </v-avatar>
  </template>
</v-chip>
```

Reply:
This was partially due to a styling issue I resolve in https://github.com/vuetifyjs/vuetify/commit/d8fd2297d04b130a484b723aa9c21dff7546c9a1

These are the supported use-cases:

1. **prepend-avatar** without a slot, we generate an avatar with the **image** and **start** props
```html
  <v-chip prepend-avatar="..." />
``` 
2. **prepend-avatar** with a slot, bring your own v-avatar
```html
  <v-chip prepend-avatar="...">
    <template #prepend>
      <v-avatar />
    </template>
  </chip>
```
3. default slot with props
```html
  <v-chip>
    <v-avatar image="..." start />
  </chip>
```
4. prepend-slot with props
```html
  <v-chip>
    <template #prepend>
      <v-avatar image="..." start />
    </template>
  </chip>
```